### PR TITLE
Citizen science

### PR DESCRIPTION
**Pull request by rosemm**
New module on citizen science (the product of me losing power during the storm for 12 hours...). NIH strategic plan for data science emphasizes citizen science and public involvement, so seemed like good low hanging fruit.

### CODE REVIEW

File: citizen_science/citizen_science.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
+<!--
+
+author:   Rose Hartman
+email:    hartmanr1@chop.edu
+version:  1.0.0
+module_template_version: 3.1.0
```
Comment (drelliche) [inline citizen_science/citizen_science.md:6]:
```suggestion
module_template_version: 3.0.0
```

File: citizen_science/citizen_science.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+For an idea of what might be involved in setting up a larger project, see the [Federal Crowdsourcing and Citizen Science Toolkit](https://www.citizenscience.gov/toolkit/#) published by the U.S. government.
+
+</div>
+
+A more reasonable option for researchers looking to get started with citizen science is to post a project on an existing platform, like [Zooniverse](https://www.zooniverse.org/lab).
+
+
+## Quiz: Citizen Science
```
Comment (drelliche) [inline citizen_science/citizen_science.md:138]:
```suggestion
## Quiz
```

Comment (drelliche) [reply citizen_science/citizen_science.md:138]: The template suggests no quiz "topic" if there is only one quiz.

File: citizen_science/citizen_science.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+Citizen science is attractive to many researchers not only because of the possibility of outsourcing work on large complex data, but because it provides the potential to democratize science and allow the public more influence over and involvement in the scientific process.
+
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+Some projects focus especially on how public involvement in science can put power back in the hands of communities.
+For example, Public Lab works to pursue environmental justice through community science and open technology. From the [Public Lab About page](https://publiclab.org/about):
+
+"When people can easily and reliably track local effects associated with environmental injustices — increased flooding, poor air quality, pollution and destruction of wetlands — they can make better-informed decisions and take action."
```
Comment (drelliche) [inline citizen_science/citizen_science.md:79]:
```suggestion
> When people can easily and reliably track local effects associated with environmental injustices — increased flooding, poor air quality, pollution and destruction of wetlands — they can make better-informed decisions and take action.
```

File: citizen_science/citizen_science.md [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+
+<div class = "history">
+<b style="color: rgb(var(--color-highlight));">Historical context</b><br>
+
+[What Is Citizen Science? – A Scientometric Meta-Analysis](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0147152) explores the history of citizen science projects, including which fields make the most use of citizen science and what scientific output they have produced.
+It includes a meta-analysis of more than 2000 published papers.
+
+</div>
+
+## Why participate in citizen science?
```
Comment (drelliche) [inline citizen_science/citizen_science.md:95]: My initial thought was that this section would be about why a scientist would want to structure something as "citizen science" but I'm not sure how to rephrase this to make it clear that this section is about the citizens volunteering.

File: citizen_science/citizen_science.md [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+
+Citizen scientists vary a tremendous amount in terms of the level of dedication they bring to a project, their expertise, and their motivation for participating.
+In some cases, participants are attracted just by the prospect of contributing to a scientific project, especially if it's an issue they care about.
+They also may be motivated by helping their community or themselves with the output of the research.
+
+
+Other times, the project is [designed as a game](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0100662) or includes more traditional incentives like money to attract participants.
```
Comment (drelliche) [inline citizen_science/citizen_science.md:102]:
```suggestion
Other times, the project includes more traditional incentives like money to attract participants, or is designed as a game. Using free smartphone games to collect research data can be quite effective, as this [article in PLOS]((https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0100662)  shows.
```

I'm not sure how to make this link both accessible (clear what sort of thing it goes to) and flow nicely, but here is a mediocre suggestion.

File: citizen_science/citizen_science.md [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+
+All of these are reasons people put their time and effort towards citizen science projects!
+
+As you learn about new citizen science projects, notice the language they use on their recruitment materials and think about what kinds of motivation they're attempting to tap into.
+If you start your own citizen science project, carefully consider what might best motivate your participants.
+
+</div>
+***
+
+True or False: There are platforms available online to help you launch new citizen science projects quickly.
```
Comment (drelliche) [inline citizen_science/citizen_science.md:174]: I was thrown by the word "quickly" given how the module had early emphasized how these projects can take a long time to get momentum to produce scientific results.

File: citizen_science/citizen_science.md [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+comment:  This is an overview of citizen science for biomedical researchers.
+long_description: This module describes what citizen science is, how it is used in research, and what motivates participants to contribute. It also highlights several examples of real citizen science projects being used in biomedical research and related fields. No prior knowledge is assumed.
+estimated_time: 30 minutes
+@learning_objectives  
+
+After completion of this module, learners will be able to:
+
+- list several ways members of the public can contribute to scientific projects
+- identify several different factors that may motivate people to participate in citizen science
+- analyze existing citizen science projects for things like policies on collaboration, goals of the research, and strategies for implementation
```
Comment (drelliche) [inline citizen_science/citizen_science.md:19]: This learning objective is definitely focused on in the quiz, but only a little discussed in the module itself. Could some of the content in the quiz be moved to the main portion? Maybe analyze one of the articles "together" and then the other one can be a quiz question.

**Review by drelliche — CHANGES_REQUESTED** [review]
This is a really informative module! I learned a lot!

There were some paragraphs and links the were a little unclear. I think perhaps renaming the sections could help a lot to differentiate/emphasize the different audience roles. What I see this module doing is:
- Defining "Citizen Science" and giving examples of it in action
- Answering "Why do people sign up to be citizen scientists?" (citizen perspective) and how do I get started in this role?
- Answering "When is crowd sourcing/citizen science right for a project?" (professional scientist perspective) and how do I get started in this role?

I think it also might be nice to include a little explainer about the difference between being a citizen scientist and being a research participant. Particularly with the studies looking at whether untrained humans or AI are better at identifying certain things in images, this question feels a little blurry.

File: citizen_science/citizen_science.md (hunk 4 above)
Comment (rosemm) [reply citizen_science/citizen_science.md:95]: Ah, good call. I changed it to "What motivates citizen scientists?"

File: citizen_science/citizen_science.md (hunk 6 above)
Comment (rosemm) [reply citizen_science/citizen_science.md:174]: Good call! Will revise
```suggestion
True or False: There are platforms available online to help you launch new citizen science projects without having to do all of the setup work yourself.
```

Comment (rosemm) [reply citizen_science/citizen_science.md]: Good call --- I revised

**Review by rosemm — COMMENTED** [review]
Okie dokie, @drelliche I tried to incorporate your suggestions --- let me know if it's still not landing. Thanks for the excellent comments!

**Comment by rosemm** [comment]
> 

Ah, sorry! I missed this comment before with the note about study participant vs citizen scientist. I added a little section in to clarify --- hopefully that helps!

**Comment by drelliche** [comment]
@rosemm I like all the updates you have made! 

I think there is a little more that could be done for the flow of the content, and I'll try to describe my thoughts here, but perhaps a quick call could help us figure out a flow?

- Would it be possible to break the quiz into 3 sections, one after each of the level 2 sections?
- Could the "analyze a real project" parts be their own subsections?

**Comment by rosemm** [comment]
@drelliche I think I like this structure better now -- let me know what you think! I also increased the time estimation, since I've added a bit of content.

File: citizen_science/citizen_science.md [hunk 8]
```diff
@@ -0,0 +1,344 @@
...
+- to access [larger and more diverse participant populations](http://cognitivehealth.tech/wp-content/uploads/2019/02/DodellFeder2019-SCZRes.pdf) than would otherwise be available through more traditional recruitment methods
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+Read [an interview with biologist Caren Cooper](https://e360.yale.edu/features/interview_caren_cooper_how_rise_of_citizen_science_is_democratizing_research) about her experience with several citizen science projects.
+She discusses how using citizen science has changed the kinds of research questions she's able to ask, and also the impact participating in science has had on the individuals and communities she works with.
+
+> In the beginning I didn’t really see the full benefits of citizen science. At first I did have this narrow, selfish view of it, like, “Oh, there are more people to help with the work and that will be great.”...
```
Comment (drelliche) [inline citizen_science/citizen_science.md:219]:
```suggestion
> In the beginning I didn’t really see the full benefits of citizen science. At first I did have this narrow, selfish view of it, like, “Oh, there are more people to help with the work and that will be great.” It didn’t take long though to learn two things. One was that I can ask these amazing questions that transcend single study sites. And the other was that I learned that these field experiences and this collaborative relationship between members of the public and scientists actually provide meaningful and really transformative experiences for individuals and for communities. 
```

I like the whole paragraph!

**Comment by drelliche** [comment]
My only suggestion is to expand the Caren Cooper quote, and then I think it is ready to go! Definitely 45 minutes seems right for this.

**Review by drelliche — APPROVED** [review]
